RenderEngine/RenderEngine/Transformer/Matrix.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace RenderEngine.Transformer
+{
+    public class Matrix
+    {
+        public static float[,] Multiply(float[,] matrix1, float[,] matrix2)
+        {
+            float[,] multiplyMatrix = new float[matrix1.GetLength(0), matrix2.GetLength(1)];
+
+            for (int row = 0; row < matrix1.GetLength(0); row++)
```

Comment:
So the class is useless to be non-static

Reply:
not understand

Reply:
The class is non-static and contains only one static method, so the class can be marked as a static